Ecs.js

```diff
@@ -0,0 +1,70 @@
+const execSync = require('child_process').execSync;
+
+module.exports = class Ecs {
+
+    /**
+     * @param {string} clusterName
+     * @param {string} awsBinary
+     */
+    constructor(clusterName, awsBinary) {
+        this._clusterName = clusterName;
+        this._awsBinary = awsBinary;
+    }
+
+    /**
+     * Restart all tasks on the cluster
+     */
+    restartAllTasks() {
+        const tasks = this._listTasks();
+        let tasksRestarted = 0;
+
+        while (tasksRestarted < tasks.length) {
+            const taskArn = tasks[tasksRestarted];
+            let taskRestarted = false;
+
+            execSync(`${this._awsBinary} ecs stop-task --cluster ${this._clusterName} --task ${taskArn}`);
+
+            while (!taskRestarted) {
+                process.stdout.write('.');
+
+                const updatedTasks = this._listTasks();
+
+                if (!this._isTaskRunning(taskArn)
+                    && updatedTasks.length === tasks.length) {
+                    process.stdout.write("\n" + `Task ${taskArn} restarted` + "\n");
+
+                    taskRestarted = true;
+                }
+            }
+
+            tasksRestarted++;
+        }
+    }
+
+    /**
+     * Get a list of all tasks on the cluster
+     *
+     * @returns {string[]} - Array of task ARNs
+     * @private
+     */
+    _listTasks() {
```

Comment:
Correct me if I'm wrong but I think our JS styleguide says not to use trailing or leading underscores (even for "private" properties)

Reply:
The mm JS style guide is currently only applicable for ES5, so I think we need to discuss this on a different occasion.